src/kw_config_loader.sh

```diff
@@ -158,11 +158,18 @@ function parse_configuration()
 # higher level setting definitions to overwrite lower level ones.
 function load_configuration()
 {
+  if [[ -v configurations && "$OVERRIDE_CONFIG" != 1 ]]; then
+    return
+  fi
+
```

Comment:
The commit message here does not describe anything... Please, update the commit message.